content/blog/exposing-an-application-using-ingress-and-tls-termination-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,528 @@
+```
+
+### Access deployed Nginx applications
+
+B﻿efore accessing the deployed Nginx applications, you need set up the domain and the subdomain name resolution. For the sample domain name *nginx.example.com*, and its subdomains, *blue.nginx.example.com* and *green.nginx.example.com*, the workstation host file has been used for DNS resolution.  
+
+Type the following commands to check that this is done correctly: 
+
+```shell
+$ host nginx.example.com
+nginx.example.com has address 10.6.115.251
+
+$ host green.nginx.example.com
+green.nginx.example.com has address 10.6.115.251
+
+$ host blue.nginx.example.com
+blue.nginx.example.com has address 10.6.115.251
+```
+
+You can then validate the Ingres TLS configuration of the deployed Nginx applications using the browser. 
+
+S﻿tart the browser and type the URL *nginx.example.com*, it will be redirected over HTTPS with the warning message *'Your connection is not private'*: 
+
+![](/img/nginx-main-warning.png)
+
+T﻿his is due to the fact that the self-signed certifcate is generated in cert-manager and configured in the K8s Ingress resource.
+
+C﻿lick *Not secure* and start the Certificate Viewer to check the certificate:
+
+![](/img/nginx-main-cert.png)
+
+C﻿lick *Proceed to nginx.example.com (unsafe)*, you then go to the Nginx *MAIN* page:
+
+![](/img/nginx-main.png)
+
+Type the URL *green.nginx.example.com* to the browser, it will be redirected over HTTPS with the same warning message *'Your connection is not private'*:   
+
+![](/img/nginx-green-warning.png)
+
+C﻿lick *Proceed to green.nginx.example.com (unsafe)*, you then go to the Nginx *GREEN* page:
+
+![](/img/nginx-green.png)
+
+T﻿he same thing occurs when you type the URL *blue.nginx.example.com* to the browser. The access will be redirected over HTTPS with the same warning message *'Your connection is not private'*:  
+
+![](/img/nginx-blue-warning.png)
+
+C﻿lick *Proceed to blue.nginx.example.com (unsafe)*, you then go to the Nginx *BLUE* page:  
+
+![](/img/nginx-blue.png)
+
+You have successfully configured the Ingress with the generated TLS c﻿ertificate and exposed the deployed applications with TLS termination. 
+
+### Conclusion
+
+This blog post provided a comprehensive guide on how to expose applications and make them accessible securely via HTTPS in a K8 cluster in HPE GreenLake for Private Cloud Enterprise. It detailed the process of configuring TLS termination on an Ingress controller, utilizing a K8s Ingress resource and a self-signed TLS certificate generated with cert-manager. While the blog post emphasized on self-signed certificates, the outlined procedure is equally applicable to any type of certificates. This flexibility allows customers to follow the steps using their own CA certificates or any commercially issued certificates for Ingress TLS termination, ensuring secure exposure of their applications in the K8s cluster over HTTPS. 
```

Comment:
Modify the 3rd sentence to read:
"While the emphasis of this post was on self-signed certificates,..."